core/services/fluxmonitorv2/orm_test.go

```diff
@@ -174,64 +174,9 @@ func TestORM_CreateEthTransaction(t *testing.T) {
 		gasLimit = uint64(21000)
 	)
 
-	orm.CreateEthTransaction(corestore.DB, from, to, payload, gasLimit, 0)
+	txm.On("CreateEthTransaction", corestore.DB, from, to, payload, gasLimit).Return(models.EthTx{}, nil).Once()
 
-	etx := models.EthTx{}
-	require.NoError(t, corestore.ORM.DB.First(&etx).Error)
+	orm.CreateEthTransaction(corestore.DB, from, to, payload, gasLimit)
 
-	require.Equal(t, gasLimit, etx.GasLimit)
-	require.Equal(t, from, etx.FromAddress)
-	require.Equal(t, to, etx.ToAddress)
-	require.Equal(t, payload, etx.EncodedPayload)
-	require.Equal(t, assets.NewEthValue(0), etx.Value)
-}
-
-func TestORM_CreateEthTransaction_OutOfEth(t *testing.T) {
```

Comment:
this one was not needed anymore?

Reply:
Nah, since it was fully abstracted into the BPTXM it didn't make a lot of sense to keep this test.